sftp-connector-test/src_test/com/axonivy/connector/sftp/test/SftpProcessTest.java

```diff
@@ -47,17 +46,9 @@
 
 	@Test
 	@Order(1)
-	public void callOpenConnection(BpmClient bpmClient) {
-		BpmElement startable = TEST_HELPER_PROCESS.elementName("openConnection()");
-		
-		SubProcessCallResult result = bpmClient.start()
-			        .subProcess(startable)
-			        .execute() // Callable sub process input arguments 
-			        .subResult();
-		
-		SftpClientService sftpClient = result.param("sftpClient", SftpClientService.class);
-		assertThat(sftpClient).isNotNull();
-		sftpClient.close();
+	public void callOpenConnection() {
```

Comment:
does the test still run in our github actions-pipeline?  Seems like I can't trigger builds manually in your repo. Maybe you have to open a non-draft PR so that the pipeline runs automatically?
However, I assume that the public/private key combo that is now in the demo is not compatible with the ubuntu host. But as said before, we shouldn't add it to GIT anyway.

Reply:
The test class is not using the public/private keys authentication but the old username/password method. so it will work on the Ubuntu environment.